DEVELOPMENT.md

```diff
@@ -101,41 +101,41 @@
 
 ## Test Running
 
--   Run all unit tests.
+-   Run all unit and integration tests.
```

Comment:
Hmmm... this is changing the default. Is there anywhere else that we night be accidentally starting to run integration tests as well? Even if we do, maybe it's not a problem.

Reply:
`bazel test //...` has always been runig what is called integration tests in this repo in addition to unit tests, so this change must be correcting a misleading documentation